[2021-01-21 12:13:59,047] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ge_dag.print_the_context 2021-01-21T11:13:52.917786+00:00 [queued]>
[2021-01-21 12:13:59,055] {taskinstance.py:826} INFO - Dependencies all met for <TaskInstance: ge_dag.print_the_context 2021-01-21T11:13:52.917786+00:00 [queued]>
[2021-01-21 12:13:59,055] {taskinstance.py:1017} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 12:13:59,056] {taskinstance.py:1018} INFO - Starting attempt 1 of 1
[2021-01-21 12:13:59,056] {taskinstance.py:1019} INFO - 
--------------------------------------------------------------------------------
[2021-01-21 12:13:59,061] {taskinstance.py:1038} INFO - Executing <Task(PythonOperator): print_the_context> on 2021-01-21T11:13:52.917786+00:00
[2021-01-21 12:13:59,065] {standard_task_runner.py:51} INFO - Started process 12999 to run task
[2021-01-21 12:13:59,073] {standard_task_runner.py:75} INFO - Running: ['airflow', 'tasks', 'run', 'ge_dag', 'print_the_context', '2021-01-21T11:13:52.917786+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ge_dag.py', '--cfg-path', '/var/folders/s2/xy3ssc8j3ks3whp2f_b8708c0000gp/T/tmpf1q78wp8']
[2021-01-21 12:13:59,074] {standard_task_runner.py:76} INFO - Job 16: Subtask print_the_context
[2021-01-21 12:13:59,112] {logging_mixin.py:103} INFO - Running <TaskInstance: ge_dag.print_the_context 2021-01-21T11:13:52.917786+00:00 [running]> on host admins-air.home
[2021-01-21 12:13:59,143] {taskinstance.py:1230} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=sara@acloudfrontier.com
AIRFLOW_CTX_DAG_OWNER=sara
AIRFLOW_CTX_DAG_ID=ge_dag
AIRFLOW_CTX_TASK_ID=print_the_context
AIRFLOW_CTX_EXECUTION_DATE=2021-01-21T11:13:52.917786+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-01-21T11:13:52.917786+00:00
[2021-01-21 12:13:59,145] {logging_mixin.py:103} INFO - /Users/sara/airflow/dags/great_expectations
[2021-01-21 12:13:59,145] {python.py:118} INFO - Done. Returned value was: None
[2021-01-21 12:13:59,151] {taskinstance.py:1135} INFO - Marking task as SUCCESS. dag_id=ge_dag, task_id=print_the_context, execution_date=20210121T111352, start_date=20210121T111359, end_date=20210121T111359
[2021-01-21 12:13:59,164] {taskinstance.py:1195} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-01-21 12:13:59,211] {local_task_job.py:118} INFO - Task exited with return code 0
